scylla/Cargo.toml

```diff
@@ -64,6 +64,7 @@ ntest = "0.8.1"
 criterion = "0.3"
 tracing-subscriber = { version = "0.3.14", features = ["env-filter"] }
 assert_matches = "1.5.0"
+rand_chacha = "0.3.1"
```

Comment:
Is a separate crate really needed?
`rand::rngs::StdRng` also implements `SeedableRng` and it's supposed to be repeatable.

Reply:
`StdRng` doc says that:
> The algorithm is deterministic but should not be considered reproducible due to dependence on configuration and possible replacement in future library versions. For a secure reproducible generator, we recommend use of the [rand_chacha](https://crates.io/crates/rand_chacha) crate directly.